service-registry/src/main/java/io/servicecomb/serviceregistry/api/registry/Microservice.java

```diff
@@ -33,6 +34,11 @@
 public class Microservice {
   private String serviceId;
 
+  private Map<String, String> framework = new HashMap<>();
+
+  @JsonProperty(value = "registerBy")
+  private String registeredBy;
```

Comment:
Why the key of the property differs from the variable name?

Reply:
ServiceCenter provider "registerBy" to us, sdk use the "registeredBy" to fit the semantics more

Reply:
That makes sense. Thanks for the explanation.

Reply:
means "owner"?
if possible, maybe SC also make the change.

Reply:
can not understand.
Microservice class can express self and others
so when your said "registerBy" or "registeredBy", there are all correct and all wrong.